styles.css

```diff
@@ -0,0 +1,335 @@
+/* ==============
+   VARIABLES
+   ============== */
+
+:root {
+  --spacing-sm: 12px;
+  --spacing-md: 24px;
+  --spacing-lg: 36px;
+
+  --transition-default: all 300ms ease-in-out;
+}
+
+/* ==============
+   UTILITY STYLES
+   ============== */
+
+.u-button-reset {
+  background: transparent;
+  border: 0;
+  cursor: pointer;
+  font-family: inherit;
+  font-size: inherit;
+  font-weight: inherit;
+  padding: 0;
+  text-align: inherit;
+}
+
+.u-list-reset {
+  list-style: none;
+  margin-top: 0;
+  margin-bottom: 0;
+  padding-left: 0;
+}
+
+/* ==============
+   BASE STYLES
+   ============== */
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: silver;
+  color: black;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica,
+    Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+  /* system font stack */
+  line-height: 1.3;
+  margin: var(--spacing-md);
+}
+
+main {
+  padding-top: 20px;
+}
+
+main a {
+  color: #444;
+}
+
+main a:hover,
+main a:focus {
+  color: black;
+}
+
+/* ==============
+   DEMO STYLES
+   ============== */
+
+.site-header {
+  position: relative;
+}
+
+.site-header h1 {
+  display: inline-block;
+  line-height: 112px;
+  margin: 0;
+  position: relative;
+  text-align: center;
+  width: 120px;
+}
+
+.site-header .logo {
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: -1;
+}
+
+.menu-icon__line {
+  background-color: silver;
+  height: 2px;
+  margin: 8px auto;
+  transition: var(--transition-default);
+  width: 42px;
+}
+
+[aria-expanded="true"] .menu-icon__line:nth-child(1) {
+  transform: translateY(10px) rotate(45deg);
+}
+
+[aria-expanded="true"] .menu-icon__line:nth-child(2) {
+  opacity: 0;
+}
+
+[aria-expanded="true"] .menu-icon__line:nth-child(3) {
+  transform: translateY(-10px) rotate(-45deg);
+}
+
+/* ==============
+   MEDIA QUERIES
+   ============== */
+
+
+
+@media (max-width: 599px) {
+  .site-nav {
+    position: absolute;
+    top: 0;
+    right: 0;
+    text-align: right;
+    z-index: 1
+  }
+}
```

Comment:
In my original [Glitch demo](https://a11y-menu-css.glitch.me/styled-menu.html), all the styles for the `.menu-trigger` and `menu-list` were inside this media query -- this allowed the button styles to not creep out when the button isn't displayed and for the nav styles to be different on mobile vs desktop.